utopian/static/utils.js

```diff
@@ -0,0 +1,71 @@
+var seconds;
```

Comment:
we should prefer let or const

https://ponyfoo.com/articles/var-let-const

Reply:
I think I took that code snippet from [SO](https://stackoverflow.com/questions/32087518/simple-countdown-timer-mmss) a few weeks ago (hence why I didn't post a contribution about adding that), so I blame them haha! Will keep it in mind in the future, though!